### ISSUE TITLE
Attachments won't open

### ISSUE THREAD

**Issue by t10ss** [HaschekSolutions/opentrashmail#36]
When I try to open any attachment I get this message: {"status":"err","reason":"File not found"} in the browser.
The file is stored in the file system in the attachments directory
Any idea how to fix?

Thanks

**Comment by geek-at** [comment]
sounds like both your problems #36 and #37 are issues with permissions. Can you try giving your monted folders 777 rights and see if it works then.

But yeah ultimately I'll have to edit the docker flow to add a special user so not everything runs as root

**Comment by t10ss** [comment]
It fixed the deleting of files, but didn't fix attachment viewing.
Still get the same result {"status":"err","reason":"File not found"} .  I'm not finding anything in the logs about it.  Maybe there is a better place I should look.

**Comment by geek-at** [comment]
Can you confirm that the attachment is actually in the storage?

If it's not try sending another mail with an attachment now that you set the permissions of the folder to 777

**Comment by t10ss** [comment]
File is in storage. I tried sending another attachment here is the listing of the directory.  The PDF file was in the directory before the chmod to 777, so it has 777 permissions.
The PNG file was received after I set the permissions of /var/www/opentrashmail/data to 777
/var/www/opentrashmail/data/test@mailsnap.com/attachments # ls -l
total 36
-rwxrwxrwx    1 root     root         29909 Jan 21 22:28 1642804128295-Protein.pdf
-rw-r--r--    1 root     root          2281 Jan 23 03:43 1642909406496-woxen-icon.png
